cmd/cassandra-reader/main.go

```diff
@@ -121,8 +121,8 @@ func loadConfig() config {
 	dbCfg := cassandra.DBConfig{
 		Hosts:    strings.Split(mainflux.Env(envCluster, defCluster), sep),
 		Keyspace: mainflux.Env(envKeyspace, defKeyspace),
-		Username: mainflux.Env(envDBUsername, defDBUsername),
-		Password: mainflux.Env(envDBPassword, defDBPassword),
+		Username: mainflux.Env(envDBUser, defDBUser),
+		Password: mainflux.Env(envDBPass, defDBPass),
```

Comment:
Maybe we should use `user` and `pass` here.

Reply:
We can't because it comes from here: "github.com/influxdata/influxdb/client/v2"

Reply:
This is Cassandra config from `readers/cassandra/init.go`.